tests/jest/IKUpload.test.jsx

```diff
@@ -205,6 +355,78 @@ describe('IKUpload', () => {
         // verify change callback
         expect(onChange.calledOnce).toEqual(true);
       });
+      test('should call onUploadStart when file input changes', () => {
+        // mount component
+        const ikUpload = mount(
+          <IKContext publicKey={publicKey} urlEndpoint={urlEndpoint} authenticationEndpoint={authenticationEndpoint} >
+            <IKUpload onError={onError} onSuccess={onSuccess} onUploadStart={onUploadStart} />
+          </IKContext>
+        );
+        // verify setup integrity
+        expect(ikUpload.html()).toEqual('<input type="file">');
+        expect(onUploadStart.called).toEqual(false);
+
+        // trigger file change and upload 
+        ikUpload.find('IKUpload').simulate('change', sampleEvent);
+
+        // verify change callback
+        expect(onUploadStart.calledOnce).toEqual(true);
```

Comment:
We should assert on the function argument. This feedback is for all test cases in this file.